src/components/ResolveDuplicateSiteButtonAndModal/ResolveDuplicateSiteButtonAndModal.js

```diff
@@ -312,10 +334,19 @@ const ResolveDuplicateSiteButtonAndModal = ({
         Resolve
       </InlineValidationButton>
       <Modal
+        testId="resolve-duplicate-site"
         title="Resolve Duplicate"
         isOpen={isResolveDuplicateModalOpen}
         onDismiss={closeResolveDuplicateModal}
-        mainContent={mainContent}
+        mainContent={
+          isResolveDuplicateModalLoading ? (
+            <ModalLoadingIndicatorWrapper>
+              <LoadingIndicator />
+            </ModalLoadingIndicatorWrapper>
```

Comment:
Its always loaders, isnt it?

Reply:
yeah, its modal loading created by AL used in Copy Sites and MRs modals.

Reply:
I mean, a lot of testing mishaps are often due to a lack of loaders or not waiting for loaders. I assumed this change was that.